code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,318 @@
+		return
+
+	if(resolved)
+		user << "Current Ahelps:"
+		for(var/datum/adminticket/T in admintickets)
+			var/ref_mob = "\ref[T.mob]"
+			usr << "<span class='adminnotice'><b><font color=red>#[T.ID] By:</font> <A HREF='?priv_msg=[T.permckey];ahelp_reply=1'>[key_name(T.permuser)]</b></A><b> Ckey:</b> [T.permckey] <b>Name:</b> [T.permuser] <b>Unique ID:</b> [T.uID]</span>"
+			usr << "	<b>Controls:</b> (<A HREF='?_src_=holder;adminmoreinfo=[ref_mob]'>?</A>) (<A HREF='?_src_=holder;adminplayeropts=[ref_mob]'>PP</A>) (<A HREF='?_src_=vars;Vars=[ref_mob]'>VV</A>) (<A HREF='?_src_=holder;subtlemessage=[ref_mob]'>SM</A>) (<A HREF='?_src_=holder;traitor=[ref_mob]'>TP</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=[ref_mob]'>FLW</A>)"
+			usr << "	<b>Message:</b> [T.msg]"
+			usr << "	<b>Handling Admin:</b> [T.admin]"
+			usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+			if(T.resolved == "No")
+				usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Resolve)</a>"
+			else
+				usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Unresolve)</a>"
+
+	else
+		user << "Current Unresolved Ahelps:"
+		for(var/datum/adminticket/T in admintickets)
+			if(T.resolved == "No")
+				var/ref_mob = "\ref[T.mob]"
+				usr << "<span class='adminnotice'><b><font color=red>#[T.ID] By:</font> <A HREF='?priv_msg=[T.permckey];ahelp_reply=1'>[key_name(T.permuser)]</b></A><b> Ckey:</b> [T.permckey] <b>Name:</b> [T.permuser] <b>Unique ID:</b> [T.uID]</span>"
+				usr << "	<b>Controls:</b> (<A HREF='?_src_=holder;adminmoreinfo=[ref_mob]'>?</A>) (<A HREF='?_src_=holder;adminplayeropts=[ref_mob]'>PP</A>) (<A HREF='?_src_=vars;Vars=[ref_mob]'>VV</A>) (<A HREF='?_src_=holder;subtlemessage=[ref_mob]'>SM</A>) (<A HREF='?_src_=holder;traitor=[ref_mob]'>TP</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=[ref_mob]'>FLW</A>)"
+				usr << "	<b>Message:</b> [T.msg]"
+				usr << "	<b>Handling Admin:</b> [T.admin]"
+				usr << "	<b>Replied To:</b> [T.active]/<b><a href='?src=\ref[T];view_logs=\ref[T]'>(LOGS)</a></b>"
+				if(T.resolved == "No")
+					usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Resolve)</a>"
+				else
+					usr << "	<b>Resolved:</b> [T.resolved] <a href='?src=\ref[T];resolve=\ref[T]'>(Unresolve)</a>"
+
+client/proc/ahelp_count(var/modifier)
+	var/amount
+	for(var/datum/adminticket/T in admintickets)
+		switch(modifier)
+			if(0)
+				if(T.resolved == "No")
+					amount++
+			if(1)
+				if(T.resolved == "Yes")
+					amount++
+			if(2)
+				amount++
+
+	return amount
+
+
+/datum/adminticket/proc/listtickets()
+	set category = "Admin Help"
+	set name = "List Adminhelps"
+	set desc = "List all current adminhelps"
+
+	if(!check_rights(R_BAN))
+		src << "<font color='red'>Error: Only administrators may use this command.</font>"
+		return
+
+	var/count = 0
+
+	for(var/datum/adminticket/T in admintickets)
```

Comment:
why don't you use your ahelp_count proc with a modifier of 2?same effect with a single line